span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/fieldgenerators/GrpcFieldsGenerator.java

```diff
@@ -318,4 +313,27 @@ protected void handleRpcResponseMetadata(String key, JaegerSpanInternalModel.Key
     grpcBuilder.getResponseBuilder().getResponseMetadataBuilder().getOtherMetadata().put(key, ValueConverter.getString(keyValue));
     grpcBuilder.getResponseBuilder().getMetadata().put(key, ValueConverter.getString(keyValue));
   }
+
+  protected void populateOtherFields(Event.Builder eventBuilder, Map<String, AttributeValue> attributeValueMap) {
+    maybeSetGrpcHostPortForOtelFormat(eventBuilder, attributeValueMap);
+    maybeSetGrpcExceptionForOtelFormat(eventBuilder, attributeValueMap);
+  }
+
+  protected void maybeSetGrpcExceptionForOtelFormat(Event.Builder builder, Map<String, AttributeValue> attributeValueMap) {
+    if (RpcSemanticConventionUtils.isRpcTypeGrpcForOTelFormat(attributeValueMap)) {
+      if (attributeValueMap.containsKey(OTelErrorSemanticConventions.EXCEPTION_TYPE.getValue())) {
```

Comment:
wrap this also in RpcSemanticConventions. So, here use the same way if preset and set.
e.g
```
Optional<String> excetionType = RpcSemanticConventionUtils.getExceptionType(attributeValueMap);
excetionType.ifPresent(s -> builder.getGrpcBuilder().getRequestBuilder().setErrorName(s));
```
Same for the message - EXCEPTION_MESSAGE

Reply:
Since this was just field I left it. Let me do it in next pr.